CHANGELOG.md

```diff
@@ -4,29 +4,34 @@
  The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
  and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.3] - 2024-08
-
-- Upgrade `axios` to mitigate [CVE-2024-39338](https://nvd.nist.gov/vuln/detail/CVE-2024-39338)
+## [2.0.0] - 2024-10-03
```

Comment:
Are we deleting v1.0.3 from the Changelog? This looks like a mistake during resolving a merge conflict?

Reply:
fixed